e2e/armadactl_test/armadactl_test.go

```diff
@@ -221,8 +221,14 @@ jobs:
 		t.Fatalf("error creating jobfile: %s", err)
 	}
 	jobPath := jobFile.Name()
-	jobFile.Write(jobData)
-	jobFile.Sync()
+	_, err = jobFile.Write(jobData)
+	if err != nil {
+		t.Fatalf("error writing jobFile: %s", err)
```

Comment:
So I followed this pattern because this entire file uses this for the test.  Should I change all instances of this in this file?  Or change just what I've changed?

Reply:
So it looks like we could use require:

```
require.NoError(t, err) which will force a failure if the test fails.